vmm/sandbox/src/bin/cloud_hypervisor/main.rs

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2022 The Kuasar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+use std::str::FromStr;
+
+use vmm_sandboxer::cloud_hypervisor::init_cloud_hypervisor_sandboxer;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    // Initialize logging
+    let mut builder = env_logger::Builder::from_default_env();
+    builder.format_timestamp_micros();
+
+    // Initialize sandboxer
+    let sandboxer = init_cloud_hypervisor_sandboxer().await?;
+
+    // If 'log_level' field isn't set in the config file, keep the log level from the default env
+    // Otherwise, set the log level configured in the config file
+    if !sandboxer.log_level().is_empty() {
+        let log_level = log::LevelFilter::from_str(sandboxer.log_level())?;
+        builder.filter_level(log_level);
+    }
+    builder.init();
```

Comment:
Should we extra the common part of logger to a new function `init_logger()` and pass variable `sandboxer.log_level()`, something like this:
```rust
...
let sandboxer = init_cloud_hypervisor_sandboxer().await?;
init_logger(sandboxer.log_level());
...
```

Reply:
the three main.rs files are too many codes in common, maybe we can make some abstraction to remove duplicated codes.